oslo/torch/nn/parallel/data_parallel/zero/fully_sharded_data_parallel.py

```diff
@@ -232,31 +220,29 @@ def _post_backward(self):
         self.heterogeneous_manager.post_iter()
 
     def grad_handle(self, p, grad):
+        self.param_op_hook.post_backward([p])
```

Comment:
What do you think about moving this line to `_post_backward` method.

Reply:
Based on the flow of method invocations, it seems that the current implementation is correct:

_module's _pre_backward -> ZeRO hook's pre_backward -> backward -> ZeRO hook's post_backward -> parameter hook (grad_handle) -> _module's _post_backward

Before grad_handle is triggered, the ZeRO hook's post_backward should appropriately mark the tensor states.

Is that correct? If you identify any errors, please let me know, and I will make the necessary adjustments.